### ISSUE TITLE
ERROR in Unexpected module 'FlexLayoutModule in /home/anubhav2000/Desktop/ngtut/node_modules/@angular/flex-layout/typings/index.d.ts' declared by the module 'AppModule in /home/anubhav2000/Desktop/ngtut/src/app/app.module.ts'. Please add a @Pipe/@Directive/@Component annotation.

### ISSUE THREAD

**Issue by Anubhav2000** [angular/flex-layout#1214]
Here are the project dependencies : 

"dependencies": {
    "@angular/animations": "~9.0.3",
    "@angular/cdk": "^9.1.0",
    "@angular/common": "~9.0.3",
    "@angular/compiler": "~9.0.3",
    "@angular/core": "~9.0.3",
    "@angular/flex-layout": "^9.0.0-beta.28",
    "@angular/forms": "~9.0.3",
    "@angular/material": "^9.1.0",
    "@angular/platform-browser": "~9.0.3",
    "@angular/platform-browser-dynamic": "~9.0.3",
    "@angular/router": "~9.0.3",
    "rxjs": "~6.5.4",
    "tslib": "^1.10.0",
    "zone.js": "~0.10.2"
  }

**Comment by adamdport** [comment]
This looks like a duplicate of #1200, which got closed for not following the template. Yours doesn't follow the template either. That being said, [I've added a reproduction repo](https://github.com/angular/flex-layout/issues/1200#issuecomment-594540444) to that closed issue and think it should be reopened. If the team wants me to create a new issue that follows the template I'd be happy to, just don't want to flood your issues with duplicates. lmk.
